tree/ntuple/v7/inc/ROOT/libdaos_mock/daos.h

```diff
@@ -238,13 +243,78 @@ int daos_obj_update(daos_handle_t oh, daos_handle_t th, uint64_t flags,
 
 
 //////////////////////////////////////////////////////////////////////////////// daos_prop.h
-
+#include <ctype.h>
 
 /** daos properties, for pool or container */
 typedef struct {
 	char		unused; // silence [-Wextern-c-compat]
 } daos_prop_t;
 
+#define DAOS_PROP_LABEL_MAX_LEN (127)
+#define DAOS_PROP_MAX_LABEL_BUF_LEN (DAOS_PROP_LABEL_MAX_LEN + 1)
+
+/**
+ * Check if DAOS (pool or container property) label string is valid.
+ * DAOS labels must consist only of alphanumeric characters, colon ':',
+ * period '.', hyphen '-' or underscore '_', and must be of length
+ * [1 - DAOS_PROP_LABEL_MAX_LEN].
+ * \param[in] label	    Label string
+ * \return	  true      Label meets length/format requirements
+ *			  false     Label is not valid length or format
+ */
+static inline bool daos_label_is_valid(const char *label)
```

Comment:
I would vote for keeping mocks simple.  Therefore, I suggest to get rid of this function; mocks do not really distinguish UUIDs from labels (any string provided by the user is just used as a key in a map).